### ISSUE TITLE
Any plans to support Ambari 2.0+?  Options for HDP 2.3 locally?

### ISSUE THREAD

**Issue by Joseph-R** [sequenceiq/docker-ambari#83]
I want to deploy HDP 2.3 to my local workstation or to a single EC2 instance in a manner that is consistent with using CloudBreak to deploy larger HDP clusters to the cloud.

Essentially this is to keep some level of consistency between development => staging => production, where our developers mirror our production environment at a smaller scale on their local workstations.

This is especially important because we always have issues hopping between major versions of HDP.  So getting that properly deployed for HDP 2.3 is essential to smoothing the transition.

I came here from [https://github.com/sequenceiq/cloudbreak/issues/444](https://github.com/sequenceiq/cloudbreak/issues/444), where it appears CloudBreak will not be supporting either a local or single-node version of HDP.  But it does not appear this project has been updated for Ambari 2.0 or 2.1, and is generally lagging behind.

Is there any plan to catch up?   Can anybody at SequenceIQ or Hortonworks point me towards the best way to accomplish that goal?

**Comment by lalyos** [comment]
master branch is: `Ambari 2.1.0 ready to install HDP 2.3.`

**Comment by lalyos** [comment]
didn't meant to close it right away, clicked the wrong button ...

**Comment by lalyos** [comment]
Actually this docker image *is* maintained. If you check the available tags at docker hub:
https://hub.docker.com/r/sequenceiq/ambari/tags/:
- 2.0.0
- 2.1.0
- 2.1.1
- 2.1.2
The issue you are referring is a really old one, probably with outdated info. Ambari 2.1.0 is absolutely supported, docker image is ready on docker hub. Now i'm closing it deliberately ;)

**Comment by Joseph-R** [comment]
Thanks, man!  I think I just followed a link to an old branch, instead of master.

Just so I'm clear, this project is intended to use Ambari in a container to deploy HDP locally on...a VM?  Using VirtualBox, I guess?  Or as separate containers also running on Docker?

**Comment by lalyos** [comment]
ambari server and agents are all running in docker containers. HDP vill be deployed by umbra agents into the same container where amber agent is running.

VirtualBox VM might be in the picture, but only for the reason that docker isn't supported natively on OSx.
You need boot2docker to run docker on a mac, which is a VirtualBox VM.

But again umbra is running in a docker container, HDP is installed in the same container.

**Comment by Joseph-R** [comment]
Gotcha.  And I just found [this](http://blog.sequenceiq.com/blog/2014/06/17/ambari-cluster-on-docker/), which I think can help guide me through getting this set up locally.  Thanks a bunch, man!

One last question before I go...  JDPA has an enterprise support agreement with HDP.  But I've found their cloudbreak support team a little "iffy".  I've been using #cloudbreak-support-public.  Other than git issues, is there a HipChat or support channel or something where I can ask questions about _this_ project?
